### ISSUE TITLE
[Terminal]: Screen Reader does not announce Confirmation message after actions executed successfully.

### ISSUE THREAD

**Issue by v-rpundir** [microsoft/terminal#12055]
### Windows Terminal version

1.12.3472.0

### Windows build number

10.0.22504.1010

### Other Software

**Test Environment:**
OS: Windows 11 Version Dev (OS Build 22504.1010)
App: Windows Terminal Preview
Screen Reader: Narrator

### Steps to reproduce

**Repro Steps:**
1. Open Windows Terminal.
2. Open Settings page using 'Ctr+,'.
3. Activate 'Actions' tab.
4. Open Narrator using 'Win + Ctrl + Enter' keys.
5. Perform all available actions present inside 'Actions' tab. 
6. Observe the issues.

**User Experience:**
Screen Reader users UX will not be good as they will not be able to know whether the actions performed successfully or not.

**Guideline Reference:**
EN 301 549 V3.2.1: 11.5.2.15 Change notification

**Attachments:**

### Expected Behavior

 Screen Reader announce Confirmation message after any actions executed successfully.
i.e. When users reset tab color, Screen Reader should announce 'Tab Color reset successfully'

### Actual Behavior

 Screen Reader does not announce Confirmation message after actions executed successfully. Issue repro with      all available actions present inside Actions tab page.

**Comment by zadjii-msft** [comment]
We should compare this with VsCode. Or even something like, `multipleActions`. Should that announce each and every sub action that was executed? There's a lot of actions that can be bound to a lot of different places. We'd have to reconcile with when we open new tabs, panes, or do anything else that already announces itself.
